PurchasesTests/SwiftSources/LocalReceiptParsing/LocalReceiptParserTests.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  LocalReceiptParserTests.swift
+//  PurchasesTests
+//
+//  Created by Andrés Boedo on 7/1/20.
+//  Copyright © 2020 Purchases. All rights reserved.
+//
+
+import Nimble
+import XCTest
+@testable import Purchases
+
+class LocalReceiptParserTests: XCTestCase {
+    func testCanInitialize() {
+        expect { LocalReceiptParser() } .notTo(raiseException())
```

Comment:
The class doesn't do anything yet so the test is only here as an example, and to make sure that testing swift files from the target works as expected.